packages/dynamodb/tests/index.test.ts

```diff
@@ -1,16 +1,125 @@
-import { AWS } from "dynamodb"
+import { DynamoDB } from "@aws-sdk/client-dynamodb"
+import { DynamoDBDocument } from "@aws-sdk/lib-dynamodb"
 import { DynamoDBAdapter } from "../src"
 import { runBasicTests } from "../../../basic-tests"
+const isTest = process.env.JEST_WORKER_ID
```

Comment:
Could you explain this?

Reply:
I removed this part, this is only usefull if you create a client that can be used in both tests and prod settings, since we create the client in the test file and not in a common one there is no need for this